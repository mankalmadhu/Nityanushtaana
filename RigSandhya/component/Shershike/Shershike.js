import React from 'react';
import { View, Text } from 'react-native';
import shershikeVinyasa from './Shershike.vinyasa';
import PropTypes from 'prop-types';

export default Shershike = (props) => (
  <View style={shershikeVinyasa.avarana}>
    <Text style={shershikeVinyasa.pathya}>
      {props.shershike}
    </Text>
  </View>
);

Shershike.propTypes = {
  shershike : PropTypes.string.isRequired
};