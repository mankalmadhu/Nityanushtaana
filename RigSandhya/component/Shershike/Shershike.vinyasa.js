import { StyleSheet } from 'react-native';
import { Font } from 'expo';

export default shershikeVinyasa = StyleSheet.create({
  avarana: {
    flex: 1,
    padding: 4,
    flexDirection: 'row',
    alignItems: 'center'
  },
  pathya: {
    marginLeft: 70,
    fontSize: 18,
    ...Font.style('ganapati'),
    color: '#ffffff'
  }
  
});