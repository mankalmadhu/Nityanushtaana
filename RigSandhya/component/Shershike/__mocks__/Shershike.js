import React from 'react';
import { Text } from 'react-native';

export default Shershike = (props) => (
  <Text>
    {props.shershike}
  </Text>
);