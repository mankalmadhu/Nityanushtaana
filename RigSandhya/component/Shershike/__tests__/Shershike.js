import 'react-native';
import React from 'react';
import Shershike from '../Shershike';

import Renderer from 'react-test-renderer';

test("Shershike Render" , () => {

   const shershike = Renderer.create(
       <Shershike shershike='TestTitle'/>
   ).toJSON();

   expect(shershike).toMatchSnapshot();

});