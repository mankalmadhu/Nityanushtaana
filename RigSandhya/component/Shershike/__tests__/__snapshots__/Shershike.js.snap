// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shershike Render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "padding": 4,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontFamily": "Ganapati",
        "fontSize": 18,
        "marginLeft": 70,
      }
    }
  >
    TestTitle
  </Text>
</View>
`;
